modules/openapi-generator/src/main/resources/dart2/api.mustache

```diff
@@ -49,24 +49,13 @@ class {{{classname}}} {
   ///
     {{/-last}}
   {{/allParams}}
-  Future<Response> {{{nickname}}}WithHttpInfo({{#allParams}}{{#required}}{{{dataType}}} {{{paramName}}},{{^-last}} {{/-last}}{{/required}}{{/allParams}}{{#hasOptionalParams}}{ {{#allParams}}{{^required}}{{{dataType}}} {{{paramName}}},{{^-last}} {{/-last}}{{/required}}{{/allParams}} }{{/hasOptionalParams}}) async {
-  {{#hasParams}}
-    // Verify required params are set.
-    {{#allParams}}
-      {{#required}}
-    if ({{{paramName}}} == null) {
-     throw ApiException(HttpStatus.badRequest, 'Missing required param: {{{paramName}}}');
-    }
-      {{/required}}
-    {{/allParams}}
-
-  {{/hasParams}}
+  Future<Response> {{{nickname}}}WithHttpInfo({{#allParams}}{{#required}}{{{dataType}}} {{{paramName}}},{{^-last}} {{/-last}}{{/required}}{{/allParams}}{{#hasOptionalParams}}{ {{#allParams}}{{^required}}{{{dataType}}}? {{{paramName}}},{{^-last}} {{/-last}}{{/required}}{{/allParams}} }{{/hasOptionalParams}}) async {
     // ignore: prefer_const_declarations
     final path = r'{{{path}}}'{{#pathParams}}
       .replaceAll({{=<% %>=}}'{<% baseName %>}'<%={{ }}=%>, {{{paramName}}}{{^isString}}.toString(){{/isString}}){{/pathParams}};
 
     // ignore: prefer_final_locals
-    Object postBody{{#bodyParam}} = {{{paramName}}}{{/bodyParam}};
+    Object{{^bodyParam}}?{{/bodyParam}} postBody{{#bodyParam}} = {{{paramName}}}{{/bodyParam}};
```

Comment:
This creates for me some errors
```dart
  Future<Response> apiV3WorkPackagesIdPatchWithHttpInfo(int id, { bool? notify, WorkPackage? workPackage, }) async {
    // ignore: prefer_const_declarations
    final path = r'/api/v3/work_packages/{id}'
      .replaceAll('{id}', id.toString());

    // ignore: prefer_final_locals
    Object postBody = workPackage;
```
`workPackage` is nullable but `postBody` is not
spec: https://raw.githubusercontent.com/wbt-solutions/openproject_dart_sdk/master/openProject_swagger.yaml

Reply:
@noordawod could you please take a look at this?

Reply:
Should be an easy fix. How would you like me to post a fix? A pr on top of your pr, or give me temporary access?

Reply:
:heavy_check_mark: